backend/index.ts

```diff
@@ -1,5 +1,40 @@
-import type { _PublicKeyAuthenticatorProtocol } from '../types/interface'
-export interface CreateAuthenticatorOptions {}
-export function createPublicKeyAuthenticator(opts: CreateAuthenticatorOptions): _PublicKeyAuthenticatorProtocol {
-    return {}
+import type { PublicKeyAuthenticatorProtocol } from '../types/interface'
+
+export interface NormalizedCreateOptions {
+    keys: CryptoKeyPair
+    timeout: number
+    rpID: string
+    challenge: BufferSource
+    crossOrigin: boolean
+}
+
+export interface CreateAuthenticatorOptions {
+    getNormalizedCreateOptions(): Promise<NormalizedCreateOptions>
+
+    // sign count
+    getSignCount(key: CryptoKey): Promise<number>
```

Comment:
The parameter should be relying party ID and credential ID

Reply:
Why? each `key` should be related to only one sign count, not RPID. and one credential ID is something same as the crypto key

Reply:
It possible to re-derive the key from RPID and Credential ID but not the reverse. We may need to split those process in the mask backend.